contents/data_vis_aog.md

```diff
@@ -0,0 +1,231 @@
+# Data Visualization with AlgebraOfGraphics.jl {#sec:aog}
+
+> **_NOTE:_**
+> Don't forget to check [`AlgebraOfGraphics.jl` documentation](https://aog.makie.org).
+
+`AlgebraOfGraphics.jl` is a higher-level plotting package that uses `Makie.jl` under the hood.
+It is geared towards data visualization workflows with support for `DataFrame`s.
+`AlgebraOfGraphics.jl` abstracts away some common patterns in data visualization using an interface that was based on the "grammar of graphics" approach that is used by R's `ggplot2` package.
+
+The **grammar of graphics** is a framework that follows a layered approach to construct visualizations in a structure manner.
```

Comment:
I don't agree with the wording **grammar**, it's too R. Here it is **Algebra**. Also, I think that it would be fitting to have a mention to https://aog.makie.org/stable/philosophy/

Reply:
Please check if [6df1648](https://github.com/JuliaDataScience/JuliaDataScience/pull/289/commits/6df164833c7e4554560c06bdb4988e77f96c9aa4) solves this.